Resources/public/js/apps/ez-platformuiapp.js

```diff
@@ -733,6 +739,13 @@ YUI.add('ez-platformuiapp', function (Y) {
                     view: 'studioPlusPresentationView',
                     callbacks: ['open', 'checkUser', 'handleSideViews', 'handleMainView']
                 }, {
+                    name: "translateContent",
```

Comment:
since you are adding a route which is almost the same as editContent, you also need to duplicate the route configuration passed to the application in shell.html.twig (will change that soon anyway...)